test/unit/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_executable(UnitTest
     ClientConfigBuilder_Test.cpp
     impl/http/IHttpClient_Test.cpp
     impl/lb/LoadBalancer_Test.cpp
+    impl/cli/Ping_Test.cpp
```

Comment:
I think it's better to sort by alphabet?

Reply:
Ditto